src/Stratis.Features.SQLiteWalletRepository/TransactionsToLists.cs

```diff
@@ -12,19 +12,17 @@ public class TransactionsToLists : TransactionsToListsBase
     {
         private readonly DBConnection conn;
         private readonly ProcessBlocksInfo processBlocksInfo;
-        private readonly Dictionary<TopUpTracker, TopUpTracker> trackers;
 
         internal TransactionsToLists(Network network, IScriptAddressReader scriptAddressReader, ProcessBlocksInfo processBlocksInfo)
             : base(network, scriptAddressReader, processBlocksInfo.TransactionsOfInterest, processBlocksInfo.AddressesOfInterest)
         {
             this.conn = processBlocksInfo.Conn;
             this.processBlocksInfo = processBlocksInfo;
-            this.trackers = new Dictionary<TopUpTracker, TopUpTracker>();
         }
 
         public override void RecordSpend(HashHeightPair block, TxIn txIn, string pubKeyScript, bool isCoinBase, long spendTime, Money totalOut, uint256 spendTxId, int spendIndex)
         {
-            this.processBlocksInfo.PrevOuts.Add(new TempPrevOut()
+            var prevOut = new TempPrevOut()
```

Comment:
This change was only here so that we can log the `prevOut` yes?

Reply:
Yes.